cmd/lncli/main.go

```diff
@@ -292,6 +292,9 @@ func main() {
 		forwardingHistoryCommand,
 	}
 
+	// Add any extra commands determined by build flags.
+	app.Commands = append(app.Commands, extraCommands()...)
```

Comment:
What do you think of making all of these sub-commands? So it would be something along the lines of: 
```
lncli autopilot enable
lncli autopilot disable
```

Reply:
TIL! Nice 🤠